rosidl_tutorials/rosidl_tutorials_msgs/msg/Contact.msg

```diff
@@ -0,0 +1,8 @@
+bool FEMALE=true
+bool MALE=false
```

Comment:
These constant should be prefixed with `GENDER_` to clarify what they are intended to be used for.